themes/src/main/resources/theme/keycloak-preview/account/resources/app/widgets/ContinueCancelModal.tsx

```diff
@@ -0,0 +1,97 @@
+/* 
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { Modal, Button, ButtonProps } from '@patternfly/react-core';
+import {Msg} from './Msg';
+ 
+/**
+ * For any of these properties that are strings, you can
+ * pass in a localization key instead of a static string.
+ */
+interface ContinueCancelModalProps {
+    buttonTitle: string;
+    buttonVairant?: ButtonProps['variant'];
```

Comment:
@ssilvert it's just a typo `buttonVairant` vs `buttonVariant`, but I believe it may cause some trouble at line 46.